features/swap/components/TransactionTips.tsx

```diff
@@ -112,8 +133,44 @@ export const TransactionTips = ({
           </Text>
         )}
       </StyledDivForRateWrapper>
-
-      <Text variant="legend">${formattedDollarValue}</Text>
+      <Inline justifyContent={'flex-end'} gap={8} css={{ cursor: 'pointer' }}>
+        {tokenB?.tokenSymbol && (
+          <Tooltip label="Price impact due to the amount of liquity available in the pool.">
+            <Inline
+              css={{
+                backgroundColor: errorThreshold
+                  ? '$backgroundColors$error'
+                  : '$backgroundColors$secondary',
+                padding: '$2 $4 $2 $2',
+                borderRadius: 4,
```

Comment:
i think this could be `borderRadius: '$2'` for consistency

Reply:
Hey for some reason `borderRadius: 4` has different result than `borderRadius: '$2'`